### ISSUE TITLE
error after pasting char ` 

### ISSUE THREAD

**Issue by djindjic** [AtomLinter/linter-eslint#44]
```bash
[8186:0318/150013:ERROR:browser_main_loop.cc(164)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
ATTENTION: default value of option force_s3tc_enable overridden by environment.
[8222:0318/150013:INFO:renderer_main.cc(211)] Renderer process started
App load time: 393ms
[8186:0318/150015:INFO:CONSOLE(18)] "activate linter-eslint", source: /home/aleksandar/.atom/packages/linter-eslint/lib/init.coffee (18)
[8186:0318/150016:INFO:CONSOLE(79)] "Window load time: 2419ms", source: file:///usr/share/atom/resources/app/static/index.js (79)
[8186:0318/150148:ERROR:channel.cc(258)] RawChannel read error (connection broken)
The program 'atom' received an X Window System error.
This probably reflects a bug in the program.
The error was 'BadWindow (invalid Window parameter)'.
  (Details: serial 793 error_code 3 request_code 40 minor_code 0)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the --sync command line
   option to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
```

**Comment by batazor** [comment]
You solved this problem?

**Comment by jfaissolle** [comment]
I wasn't able to reproduce. Anyway, from the error stack, this is most certainly an Atom problem.

**Comment by iam4x** [comment]
`ATTENTION: default value of option force_s3tc_enable overridden by environment`

seems like your env contains `force_s3tc_enable` try to unset it: `unset force_s3tc_enable`

btw, not an issue with linter. I close.
